src/retyped/jbuild

```diff
@@ -2,4 +2,4 @@
 
 (library
   ((name retyped)
-    (libraries (sedlex flow_parser))))
+    (libraries (js_of_ocaml sedlex flow_parser))))
```

Comment:
Okay! So after some experimentation I just dropped regexes and wrote my own `split` function, which at least held up to the testing from the use case. `Str` had unimplemented primitives in JS, and of course `Regexp` has them in native, so eventually I just called it. I even considered [String](https://caml.inria.fr/pub/docs/manual-ocaml/libref/String.html)'s `split_on_char`, but it's 4.04.0+, and isn't Reason not there yet?

```
A developer with problem once said, "I know! I'll use regular expressions!" Now he has two problems.

--Unknown
```